### ISSUE TITLE
COM PORTS HIDER BROKES ARDUINO

### ISSUE THREAD

**Issue by Zapcrash** [Seconb/Aimmy-Arduino-Edition#20]
Dont hide your coms port. I can use anymore my arduino because IDE can read the COM ports because it hidden, and when i click unspoof my com ports didnt back. SO make my arduinos usseless forever. Nice release

**Comment by skibidi-sigma12351** [comment]
> Dont hide your coms port. I can use anymore my arduino because IDE can read the COM ports because it hidden, and when i click unspoof my com ports didnt back. SO make my arduinos usseless forever. Nice release

you are right but stupid just reset ur arduino and it will unspoof completly and that will show ur com

**Comment by Zapcrash** [comment]
> > Dont hide your coms port. I can use anymore my arduino because IDE can read the COM ports because it hidden, and when i click unspoof my com ports didnt back. SO make my arduinos usseless forever. Nice release
> 
> you are right but stupid just reset ur arduino and it will unspoof completly and that will show ur com

I cant reset my arduino, because is a pro micro. I try reset it but it keep hided. I just bought another one and write my own spoofer without COM hidder

**Comment by Seconb** [comment]
> > > Dont hide your coms port. I can use anymore my arduino because IDE can read the COM ports because it hidden, and when i click unspoof my com ports didnt back. SO make my arduinos usseless forever. Nice release
> > 
> > 
> > you are right but stupid just reset ur arduino and it will unspoof completly and that will show ur com
> 
> I cant reset my arduino, because is a pro micro. I try reset it but it keep hided. I just bought another one and write my own spoofer without COM hidder

bro the readme literally says this only works for Leonardo, did you turn your brain off?
